src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -319,6 +363,58 @@ def test_submit_not_unique_email_different_case(self):
             "* Een gebruiker met dit e-mailadres bestaat al",
         )
 
+    def test_submit_invalid_first_name_chars_fails(self):
+        UserFactory.create(email="john@smith.com")
+        user = UserFactory.create(
+            first_name="",
+            last_name="",
+            login_type=LoginTypeChoices.digid,
+        )
+        invalid_characters = "/\"\\,.:;'"
+
+        for char in invalid_characters:
+            with self.subTest(char=char):
+                get_response = self.app.get(self.url, user=user)
```

Comment:
The name of your response variable changes in every test :D

I'd stick with just `response` unless something specific is going on and/or you really need to access multiple responses at the same time (additionally `get_response` particular reads like a callable)